lib/models/repository.dart

```diff
@@ -0,0 +1,24 @@
+class Repository {
+  final String id;
+  final String name;
+  final String fullName;
+  final String avatarURL;
+  final String license;
+
+  Repository({
+    required this.id,
+    required this.name,
+    required this.fullName,
+    required this.avatarURL,
+    required this.license,
+  });
+
+  factory Repository.fromJson(Map<String, Object> json) => Repository(
```

Comment:
For JSON we use code generation in our app. Might be an interesting improvement (not for this PR https://pub.dev/packages/json_serializable)

Reply:
sure i'll check.